G1ANT.Addon.Net/G1ANT.Addon.Net/NetAddon.cs

```diff
@@ -27,5 +26,37 @@ namespace G1ANT.Addon.Net
     [CommandGroup(Name = "vnc",  Tooltip = "A command connected with a VNC server.")]
     public class NetAddon : Language.Addon
     {
+        public override void Initialize()
+        {
+            base.Initialize();
+        }
+
+        public override void LoadDlls()
+        {
+            UnpackDrivers();
+            base.LoadDlls();
+        }
+
+        private void UnpackDrivers()
+        {
+            var unpackfolder = AbstractSettingsContainer.Instance.UserDocsAddonFolder.FullName;
+            Dictionary<string, byte[]> exeList = new Dictionary<string, byte[]>()
+            {
+                { "putty.exe", Resources.putty },
+                { "VNC.exe", Resources.VNC },
+                { "telnet.exe", Resources.telnet },
+            };
+            foreach (var exe in exeList)
+            {
+                try
+                {
+                    using (FileStream stream = File.Create(Path.Combine(unpackfolder, exe.Key)))
+                    {
+                        stream.Write(exe.Value, 0, exe.Value.Length);
+                    }
+                }
+                catch { }
```

Comment:
Why this catch is empty?

Reply:
No idea, its copy pasted from other addon.
I can only guess that we don't like to show out inconvenience :)

Reply:
Check if the files already exist and don't extract them to speed-up Robot start.
If an exception is thrown display an error to the user (RobotMessageBox) and rethrow it.

Reply:
done